frontend/src/pages/modelServing/screens/projects/ServingRuntimeModal/ServingRuntimeSizeSection.tsx

```diff
@@ -14,24 +14,21 @@ import {
   ServingRuntimeSize,
 } from '~/pages/modelServing/screens/types';
 import useGPUSetting from '~/pages/notebookController/screens/server/useGPUSetting';
-import { GpuSettingString } from '~/types';
 import ServingRuntimeSizeExpandedField from './ServingRuntimeSizeExpandedField';
 
 type ServingRuntimeSizeSectionProps = {
   data: CreatingServingRuntimeObject;
   setData: UpdateObjectAtPropAndValue<CreatingServingRuntimeObject>;
   sizes: ServingRuntimeSize[];
-  gpuSetting: GpuSettingString;
 };
 
 const ServingRuntimeSizeSection: React.FC<ServingRuntimeSizeSectionProps> = ({
   data,
   setData,
   sizes,
-  gpuSetting,
 }) => {
   const [sizeDropdownOpen, setSizeDropdownOpen] = React.useState(false);
-  const { available: gpuAvailable, count: gpuCount } = useGPUSetting(gpuSetting || 'hidden');
+  const { available: gpuAvailable, count: gpuCount } = useGPUSetting('autodetect');
```

Comment:
This is something we might wanna discuss later, last feedback I had was that GPU was always available, but we might wanna have some sort of label to enable it.